daq/faucet_states_collector.py

```diff
@@ -305,3 +311,31 @@ def get_graph(self, src_mac, dst_mac):
             self.add_link(src_mac, dst_mac, sw_2, p_2, sw_1, p_1, graph)
 
         return graph
+
+    def get_processes_info(self):
```

Comment:
keep as class method -- just overall cleaner.   Also, save information somewhere in the class so it's available. Not needed now, but at some point it'll likely be run from a timer and not on-demand, so should prepare for that eventuality.

Reply:
Done